pkg/operator/operator.go

```diff
@@ -605,10 +604,18 @@ func (op *Reporting) newPrometheusConnFromURL(url string) (prom.API, error) {
 
 	transportConfig := *kubeTransportConfig
```

Comment:
I think we want to create a new transportConfig instead of using the kube one. The kube one isn't getting us anything anymore I don't think.